clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/config/GoogleInfrastructureProviderConfig.groovy

```diff
@@ -109,6 +110,11 @@ class GoogleInfrastructureProviderConfig {
                                                           credentials,
                                                           objectMapper)
 
+        newlyAddedAgents << new GoogleHttpLoadBalancerCachingAgent(googleConfiguration.googleApplicationName(),
```

Comment:
Not yet, will do. Currently master is broken from a Chris B change from yesterday, so waiting on that to be resolved and will re-test.